### ISSUE TITLE
Add option to skip `window.onerror` setup

### ISSUE THREAD

**Issue by stream7** [airbrake/airbrake-js#198]
I'm using this library in order to notify me about errors of my third party js library so I'm not interested in all of the window's errors.

Do you think it makes sense to add an option to avoid [this](https://github.com/airbrake/airbrake-js/blob/master/src/client.coffee#L49-L50)?

**Comment by vmihailenco** [comment]
You can set `window.onerror = false` or `window.onerror = function() {}`. Does it work for you?

**Comment by stream7** [comment]
@vmihailenco I'm afraid it doesn't. My library is used by other people. I want to track my library's errors without interfering with their app's errors and error reporting.

**Comment by vmihailenco** [comment]
What do you think about https://github.com/airbrake/airbrake-js/pull/199? You should be able to use it like this:

```
var airbrake = new airbrakeJs.Client({projectId: 1, projectKey: 'abc', onerror: false});
```

**Comment by stream7** [comment]
@vmihailenco looks perfect! Thanks!

**Comment by vmihailenco** [comment]
Released v0.5.9 with new option.

**Comment by stream7** [comment]
@vmihailenco thank you!
